docs/paper/admin/reference/configuration/spigot-configuration.mdx

```diff
@@ -0,0 +1,298 @@
+      animals: 32
+      flying-monsters: 32
+      ignore-spectators: false
+      misc: 16
+      monsters: 32
+      raiders: 48
+      tick-inactive-villagers: true
+      villagers: 32
+      villagers-active-for-panic: true
+      villagers-work-immunity-after: 100
+      villagers-work-immunity-for: 20
+      wake-up-inactive:
+        animals-every: 1200
+        animals-for: 100
+        animals-max-per-tick: 4
+        flying-monsters-every: 200
+        flying-monsters-for: 100
+        flying-monsters-max-per-tick: 8
+        monsters-every: 400
+        monsters-for: 100
+        monsters-max-per-tick: 8
+        villagers-every: 600
+        villagers-for: 100
+        villagers-max-per-tick: 4
+      water: 16
+    entity-tracking-range:
+      animals: 48
+      display: 128
+      misc: 32
+      monsters: 48
+      other: 64
+      players: 48
+    growth:
+      bamboo-modifier: 100
+      beetroot-modifier: 100
+      cactus-modifier: 100
+      cane-modifier: 100
+      carrot-modifier: 100
+      cavevines-modifier: 100
+      cocoa-modifier: 100
+      glowberry-modifier: 100
+      kelp-modifier: 100
+      melon-modifier: 100
+      mushroom-modifier: 100
+      netherwart-modifier: 100
+      pitcherplant-modifier: 100
+      potato-modifier: 100
+      pumpkin-modifier: 100
+      sapling-modifier: 100
+      sweetberry-modifier: 100
+      torchflower-modifier: 100
+      twistingvines-modifier: 100
+      vine-modifier: 100
+      weepingvines-modifier: 100
+      wheat-modifier: 100
+    hanging-tick-frequency: 100
+    hopper-amount: 1
+    hopper-can-load-chunks: false
+    hunger:
+      combat-exhaustion: 0.1
+      jump-sprint-exhaustion: 0.2
+      jump-walk-exhaustion: 0.05
+      other-multiplier: 0.0
+      regen-exhaustion: 6.0
+      sprint-multiplier: 0.1
+      swim-multiplier: 0.01
+    item-despawn-rate: 6000
+    max-tick-time:
+      entity: 50
+      tile: 50
+    max-tnt-per-tick: 100
+    merge-radius:
+      exp: 3.0
+      item: 2.5
+    mob-spawn-range: 8
+    nerf-spawner-mobs: false
+    seed-ancientcity: 20083232
+    seed-buriedtreasure: 10387320
+    seed-desert: 14357617
+    seed-endcity: 10387313
+    seed-fossil: 14357921
+    seed-igloo: 14357618
+    seed-jungle: 14357619
+    seed-mansion: 10387319
+    seed-mineshaft: default
+    seed-monument: 10387313
+    seed-nether: 30084232
+    seed-ocean: 14357621
+    seed-outpost: 165745296
+    seed-portal: 34222645
+    seed-shipwreck: 165745295
+    seed-slime: 987234911
+    seed-stronghold: default
+    seed-swamp: 14357620
+    seed-trailruins: 83469867
+    seed-village: 10387312
+    simulation-distance: default
+    thunder-chance: 100000
+    ticks-per:
+      hopper-check: 1
+      hopper-transfer: 8
+    trident-despawn-rate: 1200
+    verbose: false
+    view-distance: default
+    wither-spawn-sound-radius: 0
+    zombie-aggressive-towards-villager: true
+config-version:
+  default: 12
+  description: "do not touch"
```

Comment:
on one hand maybe i should remove it so it shouldn't show at all, on the other, its then no longer a complete reference

Reply:
ive decided to keep it, with more of a description with what it does, still saying that you should not touch it